spec/arrays.dd

```diff
@@ -694,7 +724,7 @@ $(H2 $(LNAME2 bounds, Array Bounds Checking))
 
         $(P It is an error to index an array with an index that is less than
         0 or greater than or equal to the array length. If an index is
-        out of bounds, a RangeError exception is
+        out of bounds, a `RangeError` exception is
```

Comment:
This is not something that needs to be fixed in this PR necessarily, but a `RangeError` is not an `Exception`, it's an `Error` / `Throwable`. Also, currently there's a subclass of `RangeError`, `ArrayIndexError`, that gets thrown.

Reply:
Thanks. Fixed, along with the style points.